packages/components/src/components/notice/bal-modal/bal-modal-header/bal-modal-header.tsx

```diff
@@ -26,7 +26,14 @@ export class ModalHeader {
             <slot></slot>
           </h2>
         </div>
-        <bal-close class="bal-modal__header__close data-test-modal-close" onClick={this.closeHandler}></bal-close>
+        <bal-close
```

Comment:
- [ ] use a conditional rendering and not css to avoid to print to much to the DOM